pyiceberg/catalog/hive.py

```diff
@@ -651,7 +651,7 @@ def list_tables(self, namespace: Union[str, Identifier]) -> List[Identifier]:
                 for table in open_client.get_table_objects_by_name(
                     dbname=database_name, tbl_names=open_client.get_all_tables(db_name=database_name)
                 )
-                if table.parameters[TABLE_TYPE].lower() == ICEBERG
+                if table.parameters.get("table_type", "").lower() == ICEBERG
```

Comment:
```suggestion
                if table.parameters.get(TABLE_TYPE, "").lower() == ICEBERG
```
Just one minor thing: How about using constant `TABLE_TYPE` defined in `catalog/__init__.py` here? I just realized I forgot to mention this in #1258

Reply:
> Just one minor thing: How about using constant `TABLE_TYPE` defined in `catalog/__init__.py` here? I just realized I forgot to mention this in #1258

@HonahX Thanks for reviewing this! I have fixed this.